apisix/plugins/mocking.lua

```diff
@@ -0,0 +1,224 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+local ngx = ngx
+local string = string
+local type = type
+local table = table
+local math = math
+local pairs = pairs
+
+local core = require("apisix.core")
+local xml2lua = require("xml2lua")
```

Comment:
Let's put the `require` at the top, and sort all the `xxx = xxx` localized imports by their names.